src/taxonomy/taxonomy-detail/TagListTable.jsx

```diff
@@ -0,0 +1,49 @@
+import {
+  DataTable,
+  TextFilter,
+} from '@edx/paragon';
+import Proptypes from 'prop-types';
+
+import { useTagListDataResponse, useTagListDataStatus } from '../api/hooks/selectors';
+
+const TagListTable = ({ taxonomyId }) => {
+  const useTagListData = () => {
+    const { isError, isFetched } = useTagListDataStatus(taxonomyId);
+    const tagList = useTagListDataResponse(taxonomyId);
+    return { isError, isFetched, tagList };
+  };
+
+  const { tagList } = useTagListData(taxonomyId);
+
+  if (!tagList || !tagList.results) {
+    return 'Loading...';
+  }
+
+  return (
+    <DataTable
+      isPaginated
+      isFilterable
+      isSortable
+      defaultColumnValues={{ Filter: TextFilter }}
```

Comment:
From https://github.com/openedx/modular-learning/issues/138

> Do not implement Keyword filtering for now - that will come in https://github.com/openedx/modular-learning/issues/128

So please remove the filter-related settings here? We'll add them when we're ready for them.

```suggestion
      isSortable
```